rspec-core/lib/rspec/core/example_group.rb

```diff
@@ -358,6 +358,10 @@ def self.include_context(name, *args, &block)
       #
       # @see SharedExampleGroup
       def self.include_examples(name, *args, &block)
+        if block
+          RSpec.deprecate("Passing a block to `include_examples`",
+                          :replacement => "Use `it_behaves_like` instead")
```

Comment:
@JonRowe do you plan to merge this PR to the future 4.0 branch, or to 3.99 branch?
3.99 needs the `RSpec.deprecate`, while 4.0 will have a `raise` here, right?
```suggestion
          raise "Passing a block to `include_examples` is not allowed. " \
                 "Use `it_behaves_like` instead."
```

Reply:
`main` is now 4.0.0.pre, so I'd appreciate a raise here and a seperate pr adding a deprecation to 3-99-`maintenance`